core/build.gradle.kts

```diff
@@ -16,4 +16,35 @@
 
 plugins {
     `kotlin-library-convention`
-}
\ No newline at end of file
+    kotlin("multiplatform")
+}
+
```

Comment:
We should move this into the `kotlin-library-convention` file under buildSrc so it applies to both `core` and `coroutines` modules . The file defines a plugin with lots of source sets too.